src/main/java/edu/kis/powp/jobs2d/commands/OperateToCommand.java

```diff
@@ -1,31 +1,30 @@
-/*
- * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
- * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
- */
-package edu.kis.powp.jobs2d;
-
-/**
- *
- * @author student
- */
-public class OperateToCommand implements DriverCommand
-{
-    private int x;
-    private int y;
-    
-    Job2dDriver driver;

```

Comment:
Should be private!

The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).